### PR TITLE
YLP-1105 New theme

### PR DESCRIPTION
**Pull request by fatb38**

### CODE REVIEW

File: packages/yourloops/pages/hcp/patients/team-code-dialog.tsx [hunk 1]
```diff
@@ -83,15 +79,15 @@ function TeamCodeDialog(props: TeamCodeDialogProps): JSX.Element {
   const buttonsClasses = makeButtonsClasses();
   const classes = dialogClasses();
   const { code, name, onClose } = props;
-  const dialogIsOpen = code.match(reCode) !== null;
-
-  let displayCode = "";
-  if (dialogIsOpen) {
-    displayCode = code.replace(reCode, "$1 - $2 - $3");
-  }
+  const dialogIsOpen = !!code.match(reCode);
```
Comment (yvon-dblg) [inline packages/yourloops/pages/hcp/patients/team-code-dialog.tsx:82]: Not sure if theses changes was necessary

File: packages/yourloops/pages/hcp/team-member-add-dialog.tsx [hunk 2]
```diff
@@ -101,44 +86,47 @@ function AddMemberDialog(props: AddMemberDialogProps): JSX.Element | null {
...
-              checked={role === TeamMemberRole.admin}
-              onChange={handleChangeRole}
-              name="role"
-              color="primary"
-            />
-          }
-          label={t("team-add-member-dialog-checkbox-admin")}
-        />
+      <DialogContent>
+        <Box display="flex" flexDirection="column">
```
Comment (yvon-dblg) [inline packages/yourloops/pages/hcp/team-member-add-dialog.tsx:99]: The drawback adding a box, is it add an unnecessary div element to the page. But why not.

File: packages/yourloops/pages/hcp/team-member-remove-dialog.tsx [hunk 3]
```diff
@@ -118,15 +118,16 @@ function RemoveMemberDialog(props: RemoveMemberDialogProps): JSX.Element {
         <Button
           id="team-members-dialog-rmmember-button-cancel"
           onClick={handleClose}
-          className={`${dialogClasses.buttonCancel} ${buttonClasses.buttonCancel}`}
-          variant="contained">
+          variant="contained"
```
Comment (yvon-dblg) [inline packages/yourloops/pages/hcp/team-member-remove-dialog.tsx:121]: Removing the theme on all cancel buttons, change the way it was previously designed.
Was it intended in this Jira ?

File: packages/yourloops/pages/hcp/team-members-table.tsx [hunk 4]
```diff
@@ -315,30 +317,32 @@ function TeamMemberTable(props: TeamMembersProps): JSX.Element {
...
-            {t("team-member-admin")}
-          </TableCell>
-          <TableCell id={`team-members-list-${team.id}-cellheader-actions`} />
-        </TableRow>
-      </TableHead>
-      <TableBody>
-        <MembersTableBody {...props} classes={classes} />
-      </TableBody>
-    </Table>
+    <TableContainer component={Paper}>
```
Comment (yvon-dblg) [inline packages/yourloops/pages/hcp/team-members-table.tsx:320]: Why adding an upper component here ?

File: packages/yourloops/pages/patient/teams/add-dialog.tsx [hunk 5]
```diff
@@ -48,8 +48,8 @@ import TextField from "@material-ui/core/TextField";
 import metrics from "../../../lib/metrics";
 import { useTeam, Team, getDisplayTeamCode, REGEX_TEAM_CODE_DISPLAY } from "../../../lib/team";
 import diabeloopUrl from "../../../lib/diabeloop-url";
-import { makeButtonsStyles } from "../../../components/theme";
 import { AddTeamDialogContentProps } from "./types";
+import Box from "@material-ui/core/Box";
```
Comment (yvon-dblg) [inline packages/yourloops/pages/patient/teams/add-dialog.tsx:52]: Wrong import order

File: packages/yourloops/components/theme.tsx [hunk 6]
```diff
@@ -32,11 +32,12 @@ import { PaletteOptions } from "@material-ui/core/styles/createPalette";
 const commonTheme: PaletteOptions = {
   type: "light",
   primary: {
-    main: "#109182",
-    light: "#F7F7F8",
+    main: "#039be5", // green theme : #109182
```
Comment (yvon-dblg) [inline packages/yourloops/components/theme.tsx:35]: Could have been CSS var() too.

File: packages/viz/src/styles/colors.css [hunk 7]
```diff
@@ -23,7 +23,7 @@
   --text-on-basal-automated: white;
   --text-basal: #a4a4a4;
   --text-basal-automated: #3a90a4;
-  --bolus: #6FC3BB;
+  --bolus: #039be5; /* original theme : #6FC3BB */
```
Comment (yvon-dblg) [inline packages/viz/src/styles/colors.css:26]: Could have been put in CSS var() value

**Review by yvon-dblg — CHANGES_REQUESTED** [review]
- PDF images could have been moved to the theme directory too.
- Some margins / box add or changed was intentional by the original design. I'm not sure we should have changes all of theses.

File: packages/yourloops/pages/hcp/patients/team-code-dialog.tsx (hunk 1 above)
Comment (fatb38) [reply packages/yourloops/pages/hcp/patients/team-code-dialog.tsx:82]: I put the code on a diet 😅

File: packages/yourloops/pages/hcp/team-members-table.tsx (hunk 4 above)
Comment (fatb38) [reply packages/yourloops/pages/hcp/team-members-table.tsx:320]: I think all tables should have the same design for consistency, like in patients page. Mathieu approved these changes

File: packages/yourloops/pages/hcp/team-member-remove-dialog.tsx (hunk 3 above)
Comment (fatb38) [reply packages/yourloops/pages/hcp/team-member-remove-dialog.tsx:121]: Working on consistency for the whole application, Mathieu agrees with these minor changes

File: branding/diabeloop/green/logo-icon.svg [hunk 8]
```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<svg version="1.1" id="ylp-logo-icon" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 135 135" xml:space="preserve">
+<style type="text/css">
+  .ylp-icon-color { fill: #109182; }
```
[the remark this answers is not in the document]
Comment (fatb38) [reply branding/diabeloop/green/logo-icon.svg:4]: the new favicon just drop in this week !

File: branding/diabeloop/blue/logo-icon.svg [hunk 9]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <svg version="1.1" id="ylp-logo-icon" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 135 135" xml:space="preserve">
 <style type="text/css">
-  .ylp-icon-color { fill: var(--logo-primary-color, #109182); }
+  .ylp-icon-color { fill: #039be5; }
```
[the remark this answers is not in the document]
Comment (fatb38) [reply branding/diabeloop/blue/logo-icon.svg:4]: the new favicon just drop in this week !

File: packages/viz/src/styles/colors.css [hunk 10]
```diff
@@ -23,7 +23,7 @@
   --text-on-basal-automated: white;
   --text-basal: #a4a4a4;
   --text-basal-automated: #3a90a4;
-  --bolus: #6FC3BB;
+  --bolus: var(--blip-color-primary-light); /* original theme : #6FC3BB */
```
[the remark this answers is not in the document]
Comment (fatb38) [reply packages/viz/src/styles/colors.css:26]: my bad I forgot this one !

**Review by TimCanu — APPROVED** [review]

**Review by yvon-dblg — APPROVED** [review]
LGTM

**Pull request merged**